src/control/cmd/daos_server/auto_test.go

```diff
@@ -237,7 +237,8 @@ func TestDaosServer_Auto_confGen(t *testing.T) {
 				},
 			},
 			expCfg: control.MockServerCfg("ofi+psm2", exmplEngineCfgs).
-				WithNrHugePages(18432).
+				// 18 targets * 2 engines * 512 pages
+				WithNrHugePages(18 * 2 * 512).
```

Comment:
Definitely helps make it clearer, thanks. IMO creating named constants would be even better, removing the need for the comments.

Reply:
Is the change made not satisfactory? Do I need to make any further changes?